### PR TITLE
DolphinTool: support WAD verification

### PR DESCRIPTION
**Pull request by Tilka**

### CODE REVIEW

File: Source/Core/DolphinTool/VerifyCommand.cpp [hunk 1]
```diff
@@ -139,7 +139,7 @@ int VerifyCommand(const std::vector<std::string>& args)
   }
 
   // Open the volume
-  const std::unique_ptr<DiscIO::VolumeDisc> volume = DiscIO::CreateDisc(input_file_path);
+  const std::unique_ptr<DiscIO::Volume> volume = DiscIO::CreateVolume(input_file_path);
   if (!volume)
   {
     fmt::print(std::cerr, "Error: Unable to open disc image\n");
```
Comment (JosJuice) [inline Source/Core/DolphinTool/VerifyCommand.cpp:145]: This isn't quite accurate anymore. How about simply:

```suggestion
    fmt::print(std::cerr, "Error: Unable to open file\n");
```

**Review by JosJuice — APPROVED** [review]

**Review by Dentomologist — COMMENTED** [review]
The VolumeDisc.h header can be changed to Volume.h.

The help text for the input parameter says "Path to disc image FILE."; we should probably change that to something else.

**Comment by JosJuice** [comment]
You removed the include of VolumeDisc.h without adding an include of Volume.h.

**Comment by Tilka** [comment]
It's already included in VolumeVerifier.h.

**Comment by JosJuice** [comment]
Relying on indirect includes is not good practice. Who knows, maybe one day in the future VolumeVerifier.h will forward declare Volume.h?

**Pull request merged**